lib/llm/src/protocols/openai/chat_completions/delta.rs

```diff
@@ -112,6 +112,40 @@ impl DeltaGenerator {
         self.usage.prompt_tokens = isl;
     }
 
+    pub fn create_logprobs(
+        &self,
+        tokens: Vec<common::llm_backend::TokenType>,
+        logprobs: Option<common::llm_backend::LogProbs>,
+        _top_logprobs: Option<common::llm_backend::TopLogprobs>,
+    ) -> Option<async_openai::types::ChatChoiceLogprobs> {
+        if !self.options.enable_logprobs {
+            return None;
+        }
+
+        let token_logprobs = match (tokens, logprobs) {
+            (tokens, Some(logprobs)) => Some(
+                tokens
+                    .iter()
+                    .zip(logprobs)
+                    .map(
+                        |(token, lp)| async_openai::types::ChatCompletionTokenLogprob {
+                            token: token.clone().unwrap_or_default(),
+                            logprob: lp as f32,
+                            bytes: None,
+                            top_logprobs: [].to_vec(),
```

Comment:
`vec![]`

Reply:
done. and used `vec![]` in place of `Vec::new()` in other places.